data/transition-sites/dclg_localdirect.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
```

Comment:
Sorry. I thought "TNA crawl underway" meant "none yet" - should have checked! :disappointed: Anyway, done. :relieved: